src/components/inline/Button/Button.js

```diff
@@ -3,18 +3,59 @@ import React from "react";
 
 import s from "./Button.module.css";
 
-const Button = ({ children }) => (
-  <button className={s.button} type="button">
-    {children}
-  </button>
-);
+const Button = ({ button }) => {
+  let buttonView;
 
-Button.propTypes = {
-  children: PropTypes.node,
+  if (button.externalLink) {
+    buttonView = (
+      <a rel="noreferrer" target="_blank" href={button.externalLink}>
+        <button type="button" className={s.button}>
+          {button.text}
+        </button>
+      </a>
+    );
+  } else if (button.internalLink) {
+    buttonView = (
+      <a href={button.internalLink}>
+        <button type="button" className={s.button}>
+          {button.text}
+        </button>
+      </a>
+    );
+  } else {
+    buttonView = (
+      <button
+        type="button"
+        className={s.button}
+        onClick={() => {
+          button.scriptAction();
```

Comment:
For flexibility, you may want to actually capture the JavaScript event object and pass it through to the prop, which would allow someone to handle the event in their own custom action.